components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -155,14 +156,7 @@ const ContributorListTableRow = ({
         )}
 
         {/* Column: Contributors */}
-        <div className={clsx("flex-1 lg:min-w-[12.5rem] overflow-hidden")}>
-          <DevProfile
-            company={user?.company || getLastContributedRepo(data)}
-            username={login}
-            hasBorder={!contributor.author_login}
-          />
-        </div>
-
+        <AvatarHoverCard contributor={contributor.username} repositories={[]} size="small" />
```

Comment:
I'm currently not passing any repos and it still works and lists 3 repos (max) as per the hover card constraints.

I can see about adding repos in the array here, but wonder if it's necessary.

Reply:
not necessary in this context i'd say... we can ship as is